src/styles/global.css

```diff
@@ -36,6 +36,17 @@
     @apply text-sm sm:text-base;
   }
 
+  a {
+    @apply underline;
```

Comment:
This I would not do globally. It seems a bit weird that now everything is underlined. 
Do we really wan that? @karlkristopher this will for example also style the header  and footer links, which looks not so nice to me.